src/osmf/lib/HDSPlayer.as

```diff
@@ -156,9 +156,9 @@ package
      * @param {string} channelBranch It can be info, debug, warn, error or log.
      * @returns {boolean} True or false indicating success
      */
-    private function SendToDebugger(value:String, referrer:String = null, channelBranch:String = "info"):Boolean
+    private function SendToDebugger(value:String, referrer:String = null, channelBranch:String = "log"):Boolean
     {
-      var channel:String = "console." + channelBranch;
+      var channel:String = "OO." + channelBranch;
```

Comment:
Good idea. We're using that channelBranch throughout our code, but I changed all the entries to log just now. In a future fix I can just remove that argument from this function.